docs/2/technical/data-format/overview.md

```diff
@@ -114,55 +107,49 @@ Here we will see one by one the different components, starting with the unique v
 "updaterId": "yourname@microshare.io"
 ```
 
-* `"_id"`
+* `"_id"` and `"id"`
 
 The Id of the data is its unique identifier. It is used by a Microshare® Technician to find a particular piece of data in the datalake.
 
 * `"checksum"`
 
-The checksum is another identifier also for other needs for Microshare®.
+The checksum is another identifier that Microshare® technicians use to find that peice of data in the datalake.
 
 * `"createDate"`
 
-CreateDate as its name indicates quite well when this piece of data was created.
+CreateDate details the time the peice of data was created.
 
 * `"creatorId"`
 
-CreatorId indicates who created this piece of data, you will find here the email of the account that created this data.
+CreatorId suplies the email address of the creator this piece of data.
 
 * `"desc"`
 
-This value is usually empty, so you can ignore it when it is empty.
-
-* `"id"`
-
-This value is exactly the same as `"_id"`
+This object serves to describe the sensor the information comes from. Usually, this object is empty. 
 
 * `"name"`
 
-This value is usually empty, so you can ignore it when it is empty.
+This object gives the name of the sensor that the information comes from. This field is usually empty. 
 
 * `"recType"`
 
-If you are on this page it's because you have started to manipulate the Microshare® APIs and in this case you are familiar with the notion of recType, as a quick reminder the recType is the endpoint of a data flow in the data lake.
+The recType is the endpoint of a data flow in the data lake, the name underwhich your data is stored. [You can learn more about recTypes here.](/docs/2/technical/api/api-collection/#api-standards)
```

Comment:
The recType is a notation that describes the format of the data. It is used to direct the flow of processing throughout the Smart Network. It is the most important tag used in the storage and retrieval of information in the Microshare system. recTypes that begin with io.microshare are using a canonical JSON format that is managed by Microshare. You will find these canonical formats described here. [You can learn more about recTypes here.](/docs/2/technical/api/api-collection/#api-standards)